api/build.sh

```diff
@@ -1,4 +1,5 @@
 #! /bin/sh
 
+../bin/compile-xmlsec1.sh
```

Comment:
The SAML solution has an external dependency on `xmlsec1` binary being present on the system. This will compile the binary if it is not already present.